src/blocks/gallery-masonry/inspector.js

```diff
@@ -1,124 +0,0 @@
-/**
```

Comment:
Same question here. It's not needed anymore?

Reply:
These controls have been moved to the edit.js file now. I wonder which might be improved readability either to split the code or to have all of the controls within the edit function.